services/galley/src/Galley/Intra/Push.hs

```diff
@@ -112,7 +116,7 @@ data PushTo user = Push
     _pushNativePriority :: Maybe Gundeck.Priority,
     _pushAsync :: Bool,
     pushOrigin :: Maybe UserId,
-    pushRecipients :: List1 (RecipientBy user),
+    _pushRecipients :: List1 (RecipientBy user),
```

Comment:
The ones with a leading underscore get corresponding lenses thanks to the call to `makeLenses PushTo` below. We've added `pushRecipients` to the fields that have a lens, because we needed to use it when creating a push to a single client (by default `newPush` creates pushes for all clients).

There should be no problem in adding underscores to all fields (thus getting more consistent naming for fields vs lenses), but I guess it can be done in a separate PR.

Reply:
I didn't see the makeLenses call, ok!